src/routes/landing/components/Heading.js

```diff
@@ -1,20 +1,110 @@
+const API_URL_OVERRIDE_MIN_CLICKS = 10
+const CLICK_TIMEOUT = 8
+
+type StateType = {| clickCount: number, clickStart: ?Moment, apiUrlOverride: ?string |}
+
+type PropsType = {|
+  clearResourcesAndCache: () => void,
+  theme: ThemeType
+|}
+
+const ApiUrlText = styled.Text`
+padding-top: 10px;
+color: red;
+`
 
 const LocationImage: StyledComponent<{}, ThemeType, *> = styled.Image`
   height: 70px;
   resize-mode: contain;
 `
 
-class Heading extends React.Component<{||}> {
+const Wrapper: StyledComponent<{| children: React.Node |}, {}, *> = styled.View`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`
+
+class Heading extends React.Component<PropsType, StateType> {
+  constructor (props: PropsType) {
+    super(props)
+    this.state = { clickCount: 0, apiUrlOverride: null, clickStart: null }
+  }
+
+  componentDidMount () {
+    const appSettings = new AppSettings()
+    appSettings.loadApiUrlOverride().catch(() => {}).then(apiUrlOverride => this.setState({ apiUrlOverride }))
+  }
+
+  onImagePress = async () => {
+    const prevClickCount = this.state.clickCount
+    const clickStart = this.state.clickStart
+    const clickedInTimeInterval = clickStart && clickStart.isAfter(moment().subtract(CLICK_TIMEOUT, 's'))
+
+    if (prevClickCount + 1 >= API_URL_OVERRIDE_MIN_CLICKS && clickedInTimeInterval) {
+      const appSettings = new AppSettings()
+      const apiUrlOverride = await appSettings.loadApiUrlOverride()
+      const newApiUrl = (apiUrlOverride === testBaseUrl) || (!apiUrlOverride && baseUrl === testBaseUrl) ? liveBaseUrl : testBaseUrl
+
+      await appSettings.setApiUrlOverride(newApiUrl)
+      this.setState({ clickCount: 0, clickStart: null })
+      this.props.clearResourcesAndCache()
+
+      console.debug(`Switching to new API-Url: ${newApiUrl}`)
+      this.showApiUrlToast(newApiUrl)
+    } else {
+      const newClickStart = clickedInTimeInterval ? clickStart : moment()
+      const newClickCount = clickedInTimeInterval ? prevClickCount + 1 : 1
+      this.setState({ clickCount: newClickCount, clickStart: newClickStart })
+    }
+  }
+
+  showApiUrlToast = (apiUrl: string) => Toast.show(`Switched to new API-Url: ${apiUrl}`, Toast.LONG)
+
+  switchApi = async () => {
+    const appSettings = new AppSettings()
+    await appSettings.setApiUrlOverride(baseUrl)
+    this.setState({ clickCount: 0 })
+    this.props.clearResourcesAndCache()
+    this.showApiUrlToast(baseUrl)
+  }
+
+  renderApiUrlText = () => {
+    const theme = this.props.theme
+    const apiUrlOverride = this.state.apiUrlOverride
+    if (apiUrlOverride && apiUrlOverride !== baseUrl) {
+      return (
+        <>
+          <ApiUrlText>{`Currently using API: ${apiUrlOverride.toString()}`}</ApiUrlText>
+          <Button titleStyle={{ color: theme.colors.textColor }}
+                  buttonStyle={{ backgroundColor: theme.colors.themeColor, marginTop: 10 }}
+                  onPress={this.switchApi} title={'Switch back to default API'} />
```

Comment:
Not sure whether we need to translate this. I think we do not need to.

Reply:
@Schedulaar @klinzo

Reply:
Ìmo we don't need to (but we can if we wanted to^^)